registry/src/skills.json

```
@@ -7,6 +7,72 @@
         "author": null,
         "description": "Enables your agent to fetch token details for any token on Base",
         "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details",
+        "imageUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-moxie-token-details/images/logo.png"
+    },
+    {
+        "pluginId": "ba2e0746-d089-4539-a23b-84882bc8eafd",
+        "name": "@moxie-protocol/plugin-betswirl",
+        "version": "0.0.1",
+        "displayName": "Play Games of Luck",
+        "author": "Romuald Hog <romuald@betswirl.com>",
+        "description": "Enabled your agent to place bets on BetSwirl's games of luck, and retrieving bets information.",
+        "githubUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl",
+        "logoUrl": "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl/images/logo.png",
+        "settings": {},
+        "capabilities": [
+            "Flip a Coin",
+            "Roll a Dice",
+            "Spin a Roulette",
+            "Get your bets"
+        ],
+        "starterQuestions": [
+            {
+                "label": "Bet 0.01 ETH on heads",
+                "value": "Bet 0.01 ETH on heads"
+            },
+            {
+                "label": "Bet 0.01 ETH above 44",
+                "value": "Bet 0.01 ETH above 44"
+            },
+            {
+                "label": "Bet 0.01 ETH on 3,6,8 and 10",
+                "value": "Bet 0.01 ETH on 3,6,8 and 10"
+            },
+            {
+                "label": "Show my bets",
+                "value": "Show my bets"
+            }
+        ],
+        "mediaUrls": [
+            "https://github.com/moxie-protocol/moxie-agent-skills/tree/main/packages/plugin-betswirl/images/logo.png"
+        ],
+        "actions": [
+            "COIN_TOSS",
+            "COIN_FLIP",
+            "DOUBLE_OR_NOTHING",
+            "TOSS_A_COIN",
+            "BETSWIRL_COIN_TOSS",
+            "DICE",
+            "ROLL_A_DICE",
+            "DICE_ROLL",
+            "BETSWIRL_DICE",
+            "ROULETTE",
+            "SPIN_A_ROULETTE",
+            "BETSWIRL_ROULETTE",
+            "GET_BETS",
+            "RETRIEVE_BETS",
+            "SHOW_BETS",
+            "LAST_BETS",
+            "GET_BETSWIRL_BETS"
+        ],
```

adding all actions + similes to ensure proper registration
```suggestion
        "actions": [
            "TOSS_A_COIN",
            "BETSWIRL_COIN_TOSS",
            "BET_ON_HEADS",
            "BET_ON_TAILS",
            "BET_ON_COIN_FLIP",
            "BET_ON_COIN_TOSS",
            "BET_ON_HEADS_BETSWIRL",
            "BET_ON_TAILS_BETSWIRL",
            "BET_ON_CON_FLIP_BETSWIRL",
            "BET_ON_COIN_TOSS_BETSWIRL",
            "DICE",
            "ROLL_A_DICE",
            "DICE_ROLL",
            "BETSWIRL_DICE",
            "BET_ON_DICE",
            "BET_ON_DICE_BETSWIRL",
            "ROULETTE",
            "SPIN_A_ROULETTE",
            "BETSWIRL_ROULETTE",
            "BET_ON_ROULETTE",
            "BET_ON_ROULETTE_BETSWIRL",
            "GET_BETS",
            "RETRIEVE_BETS",
            "SHOW_BETS",
            "LAST_BETS",
            "GET_BETSWIRL_BETS",
            "BETSWIRL_INFO",
            "WHAT_IS_BETSWIRL",
            "HOW_DOES_BETSWIRL_WORK",
            "HOW_TO_PLAY_BETSWIRL"
        ],
```